app/api/visitors.py

```
@@ -290,12 +291,20 @@ async def register_visitor(
 
 
 @router.get("/visitors/{visitor_id}", response_model=VisitorResponse)
-def get_visitor(visitor_id: int, db: Session = Depends(get_db)):
+def get_visitor(
+    visitor_id: int,
+    db: Session = Depends(get_db),
```

Ruff (0.15.6)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)